commands-v2/ramsete/robotcontainer.py

```diff
@@ -0,0 +1,92 @@
+from commands2 import RunCommand, RamseteCommand
+from commands2.button import JoystickButton
+
+from wpilib import XboxController
+from wpilib.controller import RamseteController, PIDController
+
+from wpimath.controller import SimpleMotorFeedforwardMeters
+
+from wpilib.kinematics import ChassisSpeeds
+
+from wpilib.interfaces import GenericHID
+
+from wpimath.trajectory.constraint import DifferentialDriveVoltageConstraint
+from wpimath.trajectory import TrajectoryConfig, TrajectoryGenerator, Trajectory
+
+from wpimath.geometry import Pose2d, Rotation2d, Translation2d
+
+from subsystems.drivetrain import Drivetrain
+
+from constants import constant
+
+
+class RobotContainer:
```

Comment:
The Java example has a bunch of comments that are omitted here, it would be good to have them.

Reply:
Got em. Thanks!